creator-node/src/components/healthCheck/healthCheckComponentService.js

```diff
@@ -130,6 +131,9 @@ const healthCheck = async (
     solDelegatePublicKeyBase58 = solDelegateKeyPair.publicKey.toBase58()
   } catch (_) {}
 
+  const clusterWorkersCount = clusterUtils.getNumWorkers()
+  const isClusterEnabled = clusterUtils.isClusterEnabled()
```

Comment:
fyi `isclusterEnabled` is now exposed in new `/config_check` route, and `numWorkers` can be computed from num_cpus which is already exposed in health_check. but still useful to have since that function can change anytime cc @theoilie